pkg/ccl/backupccl/alter_backup_schedule_test.go

```diff
@@ -128,12 +128,12 @@ INSERT INTO t1 values (1), (10), (100);
 	require.Equal(t, []string{
 		"PAUSED: Waiting for initial backup to complete",
 		"@daily",
-		"BACKUP TABLE mydb.public.t1 INTO LATEST IN 'nodelocal://1/backup/alter-schedule' WITH OPTIONS (detached)",
+		"BACKUP TABLE mydb.public.t1 INTO LATEST IN '*****' WITH OPTIONS (detached)",
```

Comment:
Hm, maybe `backupccl.emitAlteredSchedule` needs to pass FmtShowPasswords ?

Reply:
I was nervous to do that, because then I think `encryption_passphrase` might be revealed? But I did add a FmtShowFullURIs flag that could work.

I'm not sure whether this output should be sanitized or not, to be honest. It seems like the fact that we're already hiding `encryption_passphrase` means we should also hide the URL, since it could contain secrets?

Reply:
Oh, maybe I'm misunderstanding how `backupccl.emitAlteredSchedule` works.

Reply:
Eh, we're showing you the schedule _that you just set_ -- scrubbing the URI completely seems wrong. Sanitizing it may or may not be right, but for the purposes of the bug you are trying to fix -- unsanitized URIs in logs if I understand correctly? -- I think this is orthogonal?

Reply:
Ok, thanks. This is helpful!

Reply:
Should be fixed now.